src/readme.html

```diff
@@ -10,17 +10,17 @@
 <h1 id="logo">
 	<a href="https://www.classicpress.net/">ClassicPress</a>
 </h1>
-<p style="text-align: center">The business-focused CMS.<br>Powerful. Versatile. Predictable.</p>
+<p style="text-align: center">The CMS for Creators.<br>Stable. Secure. Instantly Familiar.</p>
 
 <h2 id="welcome">Welcome to ClassicPress!</h2>
-<p>You&#8217;ve just become part of a growing community of people who value the firm foundation WordPress 4.9.x provides for websites everywhere. <a href="https://www.classicpress.net/democracy/">Guided by our community</a>, we can leverage our collective expertise to maintain and develop the ClassicPress platform as an excellent choice for business and professional-quality websites worldwide.</p>
-<p>Our purpose is to provide a powerful, versatile, and predictable content management system tailored to the needs of the business website market. We invite you to connect with us and participate in our <a href="#resources">thriving online community</a>.</p>
-<p style="text-align: right">&#8212; The ClassicPress Founding Committee</p>
+<p>You&#8217;ve just become part of a growing community of people who value the firm foundation WordPress 4.9.x provides for websites everywhere. <a href="https://www.classicpress.net/democracy/">Guided by our community</a>, we can leverage our collective expertise to maintain and develop the ClassicPress platform as <em>the</em> content management system of choice for professionals, hobbyists, developers, sole-traders and <em>all</em> website creators worldwide.</p>
+<p>Our purpose is to provide a stable, secure, and instantly familiar content management system driven by the needs of the community. We invite you to connect with us and <a href="#resources">add your voice to the conversation.</a>.</p>
+<p style="text-align: right">&#8212; The ClassicPress Directors</p>
 
 <h2 id="installation">Installation</h2>
 <p>For more information about installing ClassicPress, see our <a href="https://docs.classicpress.net/installing-classicpress/">installation documentation</a>.</p>
 <ol>
-	<li>Download the latest release of ClassicPress from our <a href="https://www.classicpress.net/releases/">releases page</a>.</li>
+	<li>Download the latest release of ClassicPress from our <a href="https://link.classicpress.net/releases/">releases page</a>.</li>
```

Comment:
Need to fix this URL at https://github.com/ClassicPress/subdomain-link first.

Reply:
Resolved by ClassicPress/subdomain-link#3